be1-go/hub/standard_hub/message_handling.go

```diff
@@ -280,6 +281,22 @@ func (h *Hub) handlePublish(socket socket.Socket, byteMessage []byte) (int, erro
 	signature := publish.Params.Message.Signature
 	messageID := publish.Params.Message.MessageID
 	data := publish.Params.Message.Data
+	dataBytes, ok := base64.StdEncoding.DecodeString(data)
```

Comment:
Is it normal that you use StdEncoding instead of URLEncoding ?

Reply:
Nope, forgot to change that, thanks for pointing it out.